.github/workflows/ci.yml

```diff
@@ -80,13 +86,18 @@ jobs:
           echo $PR_NUMBER > ./pr/pr_number
           echo $PR_SHA > ./pr/pr_sha
           echo $MERGE_SHA > ./pr/merge_sha
-      - uses: actions/upload-artifact@v3
+
+      - name: Upload Artifact
+        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8
         with:
           name: pr_number
           path: pr/
+
       - name: Checkout EIP Repository
         uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
-      - uses: ethereum/eipw-action@master
+
+      - name: Run EIPW
+        uses: ethereum/eipw-action@ca327b6089abbb281e4c51eda75c3bdbac0e01f8
```

Comment:
Until we're happy with the lints for `eipw`, would it be okay to keep this pointing at `master`?

It's under the `ethereum` organization, so I think the risk of supply chain attacks is a bit lower.